exampleCourse/questions/demo/autograder/python/dragDropCode_addNumpy/info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "uuid": "356308B1-25E3-44EB-B455-83A4F21371BE",
+    "title": "pl-order-blocks: Randomized parameters and externally graded",
```

Comment:
You might have to talk to @mariana for this one, she came up with these new examples

Reply:
One question does not use the leading-code file, because it does not need to import any libraries. I made that question now randomized, and the title describes that. This other question uses bumpy, and hence, the leading-code is needed. I updated the title, but we still need to make changes to the question (since we want to remove leading and trailing code as an attribute of the element.